package.json

```diff
@@ -0,0 +1,79 @@
+{
+  "name": "commercetools-nodejs-boilerplate",
+  "version": "1.0.0-beta.1",
+  "description": "Generic Boilerplate for bootstrapping NodeJS applications",
+  "keywords": [
+    "commercetools",
+    "nodejs",
+    "boilerplate",
+    "starter"
+  ],
+  "homepage": "https://github.com/commercetools/nodejs-boilerplate#readme",
+  "bugs": {
+    "url": "https://github.com/commercetools/nodejs-boilerplate/issues"
+  },
+  "license": "MIT",
+  "contributors": [
+    "Abimbola Idowu <abimbola.idowu@commercetools.de> (https://github.com/hisabimbola)",
+    "Nicola Molinari <nicola.molinari@commercetools.de>",
+    "Philipp Sporrer <philipp.sporrer@commercetools.de>",
+    "Selwyn <talk@selwyn.cc> (https://selwyn.cc/)"
+  ],
+  "main": "./lib/main",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/commercetools/nodejs-boilerplate.git"
+  },
+  "scripts": {
+    "start": "npm run watch",
+    "test": "cross-env NODE_ENV=test babel-node test/*.spec.js test/*/**spec.js | tap-spec",
+    "test:watch": "watch \"npm run coverage:html\" src test -d",
```

Comment:
Why does it run coverage? I would expect that it just runs the tests and watches for changes. Coverage should stay a separate task.